libs/utils/src/cast_vote_records.test.ts

```diff
@@ -458,3 +460,95 @@ test('getExportedCastVoteRecordIds', async () => {
     await getExportedCastVoteRecordIds(exportDirectoryPath);
   expect([...castVoteRecordIds].sort()).toEqual(['1', '2', '3']);
 });
+
+test('isCastVoteRecordWriteInValid', () => {
+  expect(
+    isCastVoteRecordWriteInValid({
+      contestId: 'contest-0',
+      optionId: 'option-0',
+    })
+  ).toBeFalsy();
+  expect(
+    isCastVoteRecordWriteInValid({
+      contestId: 'contest-0',
+      optionId: 'option-0',
+      side: 'front',
+    })
+  ).toBeTruthy();
+  expect(
+    isCastVoteRecordWriteInValid({
+      contestId: 'contest-0',
+      optionId: 'option-0',
+      text: 'Mr. Magic',
+    })
+  ).toBeTruthy();
+});
+
+test('buildCVRSnapshotBallotStyleMetadata', () => {
+  expect(buildCVRSnapshotBallotStyleMetadata(BallotType.Precinct)).toEqual({
+    Status: [CVR.CVRStatus.Other],
+    OtherStatus: '{"ballotType":"precinct"}',
+  });
+});
+
+test('getCastVoteRecordBallotType', () => {
+  const snapshot: CVR.CVRSnapshot = {
+    '@type': 'CVR.CVRSnapshot',
+    '@id': '0',
+    Type: CVR.CVRType.Modified,
+    CVRContest: [],
+  };
+
+  const cvr: CVR.CVR = {
+    '@type': 'CVR.CVR',
+    BallotStyleId: '0',
+    BallotStyleUnitId: '0',
+    BatchId: '0',
+    CreatingDeviceId: '0',
+    CurrentSnapshotId: '0',
+    ElectionId: '0',
+    UniqueId: '0',
+    BallotImage: [
+      { '@type': 'CVR.ImageData', Location: 'front.jpg' },
+      { '@type': 'CVR.ImageData', Location: 'back.jpg' },
+    ],
+    CVRSnapshot: [],
+  };
+
+  function buildMockCvr(
+    otherStatus?: string,
+    status = [CVR.CVRStatus.Other]
```

Comment:
nit: Some of the tests that use this function were a bit hard for me to grok. I think I'd have an easier time with labeled inputs:

```ts
function buildMockCvr({
  otherStatus,
  status,
}: {
  otherStatus: string,
  status: string[],
}) {
```